tools/gff2JSON/gff-to-json.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="gff-to-JSON_wrapper" name="GFF to JSON" version="0.1.1">
+    <description>
+        Generate a JSON formated Gene file from GFF format
+    </description>
+    <requirements>
+        <requirement type="package" version="2.90">perl_json</requirement>
+    </requirements>
+    <command>
+        <![CDATA[
+        perl $__tool_directory__/gff-to-json.pl 
+        #for $q in $queries
+            "${q.genome2}:${q.input2}"
+        #end for
+        > $output
+        ]]>
+    </command>
+
+    <inputs>
+        <repeat min="1" name="queries" title="Series">
+            <param name="input2" type="data" format="gff3" label="Dataset"/>
+            <param name="genome2" type="text" label="Genome name" help="Genome name without white space or special character" />
+        </repeat>
+    </inputs>
+
+    <outputs>
+         <data name="output" format="json" label="$tool.name on ${on_string}" />
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="inputFile" ftype="gff3" value="test.gff" />
+            <param name="genome1" ftype="text" value="test" />
```

Comment:
Both param `name`s do not match the ones used in `<inputs>`. I would suggest to use `input` (or `gff3_input`) and `genome` everywhere (no numbers).